cms/toolbar/toolbar.py

```diff
@@ -260,11 +262,19 @@ def get_object_draft_url(self):
 
     # Internal API
 
-    def _add_item(self, item, position):
+    def _add_item(self, item, position=None):
```

Comment:
not sure I follow on creating separate lists for the right last and left last items?
Why not place the items either on the front of the existing list or at the end? Maybe using something like ``collections.deque``?